python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -765,6 +765,47 @@ def loadSkyCircle(self, ctrCoord, radius, filterName, epoch=None):
         circularRegion = sphgeom.Circle(centerVector, sphRadius)
         return self.loadRegion(circularRegion, filterName, epoch=epoch)
 
+    def loadSchema(self, filterName):
+        """Load the schema for the reference catalog.
+
+        Parameters
+        ----------
+        filterName : `str`
+            Name of camera filter.
+
+         Returns
+        -------
+        output : `lsst.pipe.base.Struct`
+            Results struct with attributes:
+
+            ``schema``
+                Schema of the reference catalogs returned by other 'load'
+                methods.
+            ``fluxField``
+                Name of the field containing the flux associated with
+                ``filterName``.
+        """
+        if not self.refCats:
+            raise RuntimeError("No reference tables could be found.")
+        cat = self.refCats[0].get()
```

Comment:
This probably flags my naivety more than anything else, but unless you feel it's obvious to everyone else, I wonder whether it'd be useful to comment on why it's ok to only use the first refcat in the list.

Reply:
It's because all of the refcat shards for the same catalog should have the same schema.  I'll add a comment.